CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 - Company Application Registraion Detail Overlay
   - UI improvements
   - Add correct registration application checklist items
+- Company Role
+  - enabled agreement handling by adding frontend logic and UI handling of optional agreements
```

Comment:
you added this change to the bugfix section even though the PR title and issue #584 suggest feature, what is correct?

Reply:
@evegufy updated changelog to feature and regarding milestone, I checked with Julia and confirmed to create PR against release/v2.1.0-RC1